tests/Common/EventsTest.php

```diff
@@ -327,17 +327,17 @@ public function testInvalidSearchSyntaxUserError($query): void
     public function invalidQueries()
     {
         return [
-            [
+            'colon as prefix' => [
                 ': success',
             ],
-            [
+            'colon as suffix' => [
                 'success:',
             ],
-            [
+            'slash as prefix' => [
                 '/GET',
             ],
-            [
-                '*tables*',
+            'asterisk as prefix+suffix' => [
+                '*tables*', // works fine with ElasticSearch 7
             ],
```

Comment:
- pro ES 5 a 6 - to [nefunguje](https://www.elastic.co/guide/en/elasticsearch/reference/5.6/query-dsl-wildcard-query.html)
- od ES 7 - to zacalo [fungovat](https://www.elastic.co/guide/en/elasticsearch/reference/7.0/query-dsl-wildcard-query.html) (resp. od 6.8)